tf-modules/distribution/main.tf

```diff
@@ -145,14 +145,18 @@ resource "aws_lambda_function" "s3_credentials" {
 
   environment {
     variables = {
+      AccessTokensTable              = aws_dynamodb_table.access_tokens[0].id
+      CMR_ACL_BASED_CREDENTIALS      = var.cmr_acl_based_tokens
```

Comment:
Can we rename `cmr_acl_based_tokens` to `cmr_acl_based_credentials` for consistency?

Reply:
yes, good idea.

Reply:
9c7b5e458c